### PR TITLE
Add types to query context on PgError

### LINKED ISSUE #40: queryContext is `any` type
The `TinyPgError` has a `queryContext` attribute which is currently an `any` type.

Is it possible to type that more strongly?  I'm not seeing any related definition in `@types/pg`.

Understood if this is an exercise best left to the library user.

### PR DESCRIPTION
**Pull request by stephenreddek**
Fixes #40

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,16 @@
 {
   "name": "tinypg",
-  "version": "7.0.1",
+  "version": "8.0.0",
   "description": "Easy way to call sql files using postgres.",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "scripts": {
-    "build-test": "npm run clean && tsc -p src/test",
```
Comment (dynajoe) [inline package.json:8]: intentionally delete the clean step?

Comment (stephenreddek) [reply package.json:8]: I can put it back. I was trying to figure out why the CI was failing. Turns out it has to do with TS and lodash types.

**Comment by stephenreddek** [comment]
Alright. The issue was something in the Lodash types was unable to resolve in the tests given the test tsconfig. I just set it to strict, fixed the issues, and the lodash compile issue went away. should be good to go if the test changes are approved.

**Comment by stephenreddek** [comment]
@dynajoe Do you want to merge and release this?

**Comment by slifty** [comment]
Wanted to follow up here since it's been a little while!  Would be fun to have this fix in.

**Comment by stephenreddek** [comment]
@dynajoe Can you release this change?

**Comment by alexmunda** [comment]
> @dynajoe Can you release this change?

Hi Stephen

**Comment by stephenreddek** [comment]
> > @dynajoe Can you release this change?
> 
> Hi Stephen

👋

Do you know how to kick off releases?

**Pull request merged**